Tribler/Core/CreditMining/CreditMiningManager.py

```diff
@@ -102,15 +102,15 @@ def add_sources(future):
         self.session_ready = Future()
         self.session_ready.add_done_callback(add_sources)
 
-    async def shutdown(self, remove_downloads=False):
+    async def shutdown(self, removes=False):
```

Comment:
Why rename this variable?

Reply:
Oops. That's one of the consequences of, seemingly, a bit too broad `sed` editing that I had to do on the codebase.